src/etwlogger_driver.c

```diff
@@ -277,50 +145,61 @@ static void perform_EventWriteLogVerboseEvent(const char* message, const char* f
     );
 
 #if CALL_CONSOLE_LOGGER
-    consolelogger_log(AZ_LOG_VERBOSE, NULL, NULL, 0, LOG_LINE, "%s", message);
+    consolelogger_log(AZ_LOG_VERBOSE, file, func, line, LOG_LINE, "%s", message);
```

Comment:
>file [](start = 38, length = 4)

We don't do this for Info above (maybe we should?). I think you want it in both or neither

Reply:
Yes, good catch! More info > less info. 

---
In reply to: [453749301](https://github.com/Azure/azure-c-logging/pull/11#discussion_r453749301) [](ancestors = 453749301)